spec/unit/grpc_web/server/message_serialization_spec.rb

```diff
@@ -16,6 +16,7 @@
     let(:service) { TestHelloService.new }
     let(:service_method) { :SayHello }
     let(:accept) { '*/*' }
+    let(:metadata) { {} }
```

Comment:
Should this be getting added to the request above? It doesn't seem to be breaking but I figured I'd check as the variable isn't referenced anywhere from what I'm seeing. I might be missing if it's somewhere in a shared test though

Reply:
Hmm good point I'm not too sure why this was added, I'll be removing it